### PR TITLE
ENH: Added more test cases for IEC transform logic

### PR DESCRIPTION
**Pull request by markgerencser**
Added test cases for gantry, collimator, patient support and table top (vertical, longitudinal, lateral) matrix checks, plus minor indentation fixes

### CODE REVIEW

**Comment by cpinter** [comment]
@ferdymercury

File: Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx [hunk 1]
```diff
@@ -141,48 +141,91 @@ int vtkSlicerIECTransformLogicTest1(int vtkNotUsed(argc), char* vtkNotUsed(argv)
...
-    {
+  {
     std::cerr << __LINE__ << ": Beam transform does not match baseline for translated isocenter" << std::endl;
     return EXIT_FAILURE;
-    }
+  }
 
-  // Gantry angle, 1 degree
-  beamNode->SetGantryAngle(1.0);
+  // Gantry angle, "-1" degree
```
Comment (cpinter) [inline Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx]: No need for the quotes. Here or later ("90"), these are not strings but numbers.

File: Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx [hunk 2]
```diff
@@ -244,19 +337,381 @@ int vtkSlicerIECTransformLogicTest1(int vtkNotUsed(argc), char* vtkNotUsed(argv)
...
+  if (!IsTransformMatrixEqualTo(mrmlScene,
+    iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::PatientSupportRotation, vtkSlicerIECTransformLogic::FixedReference),
+    expectedPatientSupportRotationToFixedReferenceTransformMinus1_MatrixElements))
+  {
+    std:cerr << __LINE__ << ": PatientSupportRotationToFixedReference does not match baseline" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  double expectedBeamTransform_PatientSupportMinus1_MatrixElements[16] =
+  { 0.0174524, 0, -0.999848, 1000,   0, 1, 0, 200,   0.999848, 0, 0.0174524, 0,   0, 0, 0, 1 };
```
Comment (cpinter) [inline Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx]: Please add some indentation here, like two spaces, so it's obvious that this is the continuation of the previous line. All other places like this too please.

File: Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx [hunk 3]
```diff
@@ -244,19 +337,381 @@ int vtkSlicerIECTransformLogicTest1(int vtkNotUsed(argc), char* vtkNotUsed(argv)
...
+  { 1, 0, 0, 230,   0, 1, 0, 0,   0, 0, 1, 0,   0, 0, 0, 1 };
+  if (!IsTransformMatrixEqualTo(mrmlScene,
+    iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::TableTop, vtkSlicerIECTransformLogic::TableTopEccentricRotation),
+    expectedTableTopToTableTop_Lateral_230_MatrixElements))
+  {
+    std::cerr << __LINE__ << ": PatientToTableTopTransform (lateral) does not match baseline" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  // Table Top (lateral), -230.0mm
```
Comment (cpinter) [inline Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx:695]: Extend comment explaining why we chose this number (that these are the min/max for the default Varian machine)

**Comment by ferdymercury** [comment]
LGTM, thanks!!

**Pull request merged**